spec.html

```diff
@@ -48962,88 +48934,89 @@ <h1>agent-order</h1>
 
     <emu-clause id="sec-reads-bytes-from" aoid="reads-bytes-from">
       <h1>reads-bytes-from</h1>
-      <p>For a candidate execution _execution_, _execution_.[[ReadsBytesFrom]] is a mathematical function mapping events in SharedDataBlockEventSet(_execution_) to Lists of events in SharedDataBlockEventSet(_execution_) that satisfies the following conditions.</p>
+      <p>For a candidate execution _execution_, its reads-bytes-from function is a mathematical function mapping events in SharedDataBlockEventSet(_execution_) to Lists of events in SharedDataBlockEventSet(_execution_) that satisfies the following conditions.</p>
```

Comment:
Did you mean to `<dfn>` this?

```suggestion
      <p>For a candidate execution _execution_, its <dfn>reads-bytes-from</dfn> function is a mathematical function mapping events in SharedDataBlockEventSet(_execution_) to Lists of events in SharedDataBlockEventSet(_execution_) that satisfies the following conditions.</p>
```

Reply:
No, it [needs to be an AOID](https://github.com/tc39/ecma262/pull/3321#discussion_r1591753761) so that ecmarkup will recognize it as valid to invoke as `reads-bytes-from(foo)`.

Reply:
Ah, okay, then let's add that `<em>`.